include/LAGraphX.h

```diff
@@ -777,6 +777,28 @@ int LAGraph_scc (
     char *msg
 ) ;
 
+//****************************************************************************
+LAGRAPHX_PUBLIC
+int LAGraph_RegularReachability // nodes reachable from the starting by the
+                                // path satisfying regular expression
+(
+    // output:
+    GrB_Vector *reachable,      // reachable(i) = true if node i is reachable
+                                // from one of the starting nodes
+    // input:
+    LAGraph_Graph *R,           // input non-deterministic finite automaton
+                                // adjacency matrix decomposition
+    size_t nl,                  // total label count, # of matrices graph and
+                                // NFA adjacenc ymatrix decompositions
+    const GrB_Index *QS,        // starting states in NFA
+    size_t nqs,                 // number of starting states in NFA
+    const GrB_Index *QF,        // final states in NFA
+    size_t nqf,                 // number of final states in NFA
+    LAGraph_Graph *G,           // input graph adjacency matrix decomposition
+    const GrB_Index *S,         // source vertices to start searching paths
+    size_t ns,                  // number of source vertices
+    char *msg
```

Comment:
What is it?

Reply:
I've followed other definitions to remain consistent when I've been writing it. 
Applied adding description.

Reply:
Resolving, since discussed verbally Semyon Vyacheslavovich is ok.